src/SliderWidget.js

```diff
@@ -59,26 +72,29 @@ export default class SliderWidget extends Component {
   };
 
   okPress (valueSubmit) {
-
-    let notification = this.state.notification;
-
-    const notification_object = this.state.notificationObject;
-    const survey_object = this.state.notification.survey;
+    let notification_object = null;
+    if(this.state.notification){
+      let notification = this.state.notification;
+      notification_object = this.state.notificationObject;
```

Comment:
it seems like we have two notification objects, "notification" and "notificationObject."  we may want to clean up the terminology and just use one.  Not necessary to make the change now but we should probably keep it in mind for future cleanup.